packages/cozy-stack-client/src/getIconURL.spec.js

```diff
@@ -1,20 +1,28 @@
 import { getIconURL } from './getIconURL'
+import getIconUrlDefault from './getIconURL'
 
 const FakeBlob = (data, options) => {
   return { data, ...options }
 }
 
+const resetcreateObjectURL = () => {
+  global.URL.createObjectURL = undefined
+  global.URL.createObjectURL = jest.fn(blob => {
+    return blob
+  })
```

Comment:
Good question it's been a long time since I wrote the code. I had a few issues with createObjectURL since this method doesn't exist in jsDom... But since I'm mocking it directly, I think I can remove it :)